### PR TITLE
Apple.BonjourPrintServices version 2.0.2.0

### PR DESCRIPTION
**Pull request by upintheairsheep**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/98682)

### CODE REVIEW

**Comment by stephengillie** [comment]
I'm not entirely sure, but based on one of the log messages I'm guessing `Microsoft.DotNet.Framework.DeveloperPack_4` might be a dependency.

File: manifests/a/Apple/BonjorPrintSerices/2.0.2.0/Apple.BonjorPrintSerices.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Apple.BonjorPrintSerices
```
Comment (Trenly) [inline manifests/a/Apple/BonjorPrintSerices/2.0.2.0/Apple.BonjorPrintSerices.locale.en-US.yaml:4]:
```suggestion
PackageIdentifier: Apple.BonjourPrintSerices
```

File: manifests/a/Apple/BonjorPrintSerices/2.0.2.0/Apple.BonjorPrintSerices.yaml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.4.0.schema.json
+
+PackageIdentifier: Apple.BonjorPrintSerices
```
Comment (Trenly) [inline manifests/a/Apple/BonjorPrintSerices/2.0.2.0/Apple.BonjorPrintSerices.yaml:4]:
```suggestion
PackageIdentifier: Apple.BonjourPrintSerices
```

File: manifests/a/Apple/BonjorPrintSerices/2.0.2.0/Apple.BonjorPrintSerices.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Apple.BonjorPrintSerices
+PackageVersion: 2.0.2.0
+PackageLocale: en-US
+Publisher: Apple Inc.
+PackageName: BonjourPrintServices
```
Comment (Trenly) [inline manifests/a/Apple/BonjorPrintSerices/2.0.2.0/Apple.BonjorPrintSerices.locale.en-US.yaml:8]:
```suggestion
PackageName: Bonjour Print Services
```

**Review by Trenly — CHANGES_REQUESTED** [review]

**Comment by stephengillie** [comment]
Manifest validation error:
> Manifest: The manifest file path must match PackageIdentifier and PackageVersion properties from manifest. actual 'manifests\a\Apple\BonjorPrintSerices\2.0.2.0', expected 'manifests\a\Apple\BonjourPrintSerices\2.0.2.0'

It's failing because the file paths are still missing the U in `Bonjour`. Also, just noticing that both are missing the V in `Services`. This implies the need for a CamelCaseSpellChecker.

File: manifests/a/Apple/BonjorPrintSerices/2.0.2.0/Apple.BonjorPrintSerices.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Apple.BonjourPrintSerices
```
Comment (stephengillie) [inline manifests/a/Apple/BonjorPrintSerices/2.0.2.0/Apple.BonjorPrintSerices.installer.yaml:4]:
```suggestion
PackageIdentifier: Apple.BonjourPrintServices
```

Comment (stephengillie) [reply manifests/a/Apple/BonjorPrintSerices/2.0.2.0/Apple.BonjorPrintSerices.installer.yaml:4]: Please update file paths to match this change.

File: manifests/a/Apple/BonjorPrintSerices/2.0.2.0/Apple.BonjorPrintSerices.locale.en-US.yaml [hunk 5]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Apple.BonjourPrintSerices
```
Comment (stephengillie) [inline manifests/a/Apple/BonjorPrintSerices/2.0.2.0/Apple.BonjorPrintSerices.locale.en-US.yaml:4]:
```suggestion
PackageIdentifier: Apple.BonjourPrintServices
```

Comment (stephengillie) [reply manifests/a/Apple/BonjorPrintSerices/2.0.2.0/Apple.BonjorPrintSerices.locale.en-US.yaml:4]: Please update file paths to match this change.

File: manifests/a/Apple/BonjorPrintSerices/2.0.2.0/Apple.BonjorPrintSerices.yaml [hunk 6]
```diff
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.4.0.schema.json
+
+PackageIdentifier: Apple.BonjourPrintSerices
```
Comment (stephengillie) [inline manifests/a/Apple/BonjorPrintSerices/2.0.2.0/Apple.BonjorPrintSerices.yaml:4]:
```suggestion
PackageIdentifier: Apple.BonjourPrintServices
```

Comment (stephengillie) [reply manifests/a/Apple/BonjorPrintSerices/2.0.2.0/Apple.BonjorPrintSerices.yaml:4]: Please update file paths to match this change.

**Comment by Trenly** [comment]
Close with reason: Stale PR;

**Pull request closed without merging**